azure-pipelines.yml

```diff
@@ -4,11 +4,46 @@ pr:
 pool:
   vmImage: 'windows-latest'
 
-steps:
-- task: DotNetCoreCLI@2
-  inputs:
-    command: 'build'
-    projects: src/SarifWeb.sln
-    configuration: 'Release'
+resources:
+  repositories:
+    - repository: 1esPipelines
+      type: git
+      name: 1ESPipelineTemplates/1ESPipelineTemplates
+      ref: refs/tags/release
+extends:
+    template: v1/1ES.Unofficial.PipelineTemplate.yml@1esPipelines
+    parameters:
+      pool:
+        name: 1ES-Shared-Hosted-Pool_Windows-Server-2022
+        os: windows
+      containers:
+        default_windows_container:
+          image: 1espipelinesdevpme.azurecr.io/windows/ltsc2022/vse2022:latest
+          registry: 1espipelinesdevpme.azurecr.io
+          tenantId: PME
+          identityType: 1ESPipelineIdentity
+        
+      stages:
+      - stage: build
+        jobs:
+          - job: build
+            templateContext:
+              inputs:
+              - input: checkout
+                repository: self
+                displayName: build
+              outputs:
+              - output: pipelineArtifact
+                targetPath: $(System.DefaultWorkingDirectory)
 
-- task: ComponentGovernanceComponentDetection@0
+            steps:
+              - task: UseDotNet@2
```

Comment:
you are using container that should have dotnet. it doesn't seem right to install it one more time

Reply:
removed